### PR TITLE
MEN-23 Added welcome page

### PR DESCRIPTION
**Pull request by mkowalc7**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,7938 @@
+{
```
Comment (luksmi) [inline package-lock.json]: Is this file necessary at all? It seems as if you've ran `npm install` command in project's root folder which generated it. All the contents related to frontend app are in menelaos-front directory, so, IMO that's the only place for package-lock.json. Please elaborate on that.

File: .gitignore [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 # dependencies
 /menelaos-front/node_modules
+/node_modules
```
Comment (luksmi) [inline .gitignore]: see comment on package-lock.json in project's root directory

File: menelaos-front/src/app/footer/footer.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (luksmi) [inline menelaos-front/src/app/footer/footer.component.spec.ts]: As discussed earlier, all *spec.ts files can be safely deleted

File: menelaos-front/src/app/app-routing.module.ts [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
 import { Routes, RouterModule } from '@angular/router';
+import { WelcomePageComponent } from "./welcome-page/welcome-page.component";
+import { RegisterComponent } from "./register/register.component";
+import { LoginComponent } from "./login/login.component";
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: '', component: WelcomePageComponent },
+  { path: 'login', component: LoginComponent },
+  { path: 'register', component: RegisterComponent }
```
Comment (luksmi) [inline menelaos-front/src/app/app-routing.module.ts]: 'about' path missing

File: package-lock.json (hunk 1 above)
Comment (mkowalc7) [reply package-lock.json]: Deleted this file.

File: .gitignore (hunk 2 above)
Comment (mkowalc7) [reply .gitignore]: fixed

File: menelaos-front/src/app/app-routing.module.ts (hunk 4 above)
Comment (mkowalc7) [reply menelaos-front/src/app/app-routing.module.ts]: added empty component

File: menelaos-front/src/app/footer/footer.component.spec.ts (hunk 3 above)
Comment (mkowalc7) [reply menelaos-front/src/app/footer/footer.component.spec.ts]: deleted all, added *.spec.ts to gitignore

**Review by luksmi — APPROVED** [review]
Looking good
